src/directives/formly-field.test.js

```diff
@@ -799,6 +812,51 @@ describe('formly-field', function() {
       expect(field.formControl.$touched).to.be.false;
       expect(field.formControl.$dirty).to.be.false;
     });
+
+    it.only(`should reset the form state for an field with multiple ng-models`, () => {
```

Comment:
Please take the `.only` off of this. Also while you're fixing that, it should be: "...for a field with..."